build.ps1

```diff
@@ -240,10 +240,12 @@ $CAKE_EXE_INVOCATION = if ($IsLinux -or $IsMacOS) {
     "`"$CAKE_EXE`""
 }
 
-
 # Build Cake arguments
-$cakeArguments = @("$Script");
-if ($Target) { $cakeArguments += "-target=$Target" }
+$cakeArguments = ""
```

Comment:
This PR changes `$cakeArguments` from an array to a string. This means that the lines below are concatenating instead of adding items to the array. It's causing problems because `.\build1.ps1 -target Foo -configuration Debug` is turned into `-target="Foo"-configuration=Debug` (note, no space), resulting in this error message:

> The target 'package-configuration=Debug' was not found.

The fix is to restore the line from the incorrect `$cakeArguments = ""` to `$cakeArguments = @()`.

This cause